src/js/grapl-cdk/destroy_all.sh

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env bash
+for BUCKET in $(aws s3 ls s3:// | cut -d' ' -f 3)
```

Comment:
This will delete all buckets in the account, right? We can't do that.

I think if we want to support this script, we should either parse specific buckets created from templates in cdk.out/ directory, or build from new deployName param.

Reply:
I figured the use-case for this was to completely blow away the entire account, e.g. to clean up our sandbox accounts. I can make it a little less heavy handed by doing a bit of filtering though.

Reply:
Punting on this for now, it's going to be more involved than is warranted right now.